src/main/java/org/folio/bulkops/domain/converter/LoanTypeConverter.java

```diff
@@ -1,11 +1,17 @@
 package org.folio.bulkops.domain.converter;
 
 import org.folio.bulkops.domain.bean.LoanType;
+import org.folio.bulkops.service.ItemReferenceHelper;
 
 public class LoanTypeConverter extends BaseConverter<LoanType> {
 
   @Override
   public String convertToString(LoanType object) {
     return object.getName();
   }
+
+  @Override
+  public LoanType convertToObject(String value) {
+    return ItemReferenceHelper.service().getLoanTypeByName(value);
```

Comment:
TBC - Looks, there is no flow to restore LoanType from csv representation to object. Why do we need this method?